src/OpenTelemetry.Api/Trace/Tracer.cs

```diff
@@ -58,14 +58,14 @@ public TelemetrySpan CurrentSpan
         /// </summary>
         /// <param name="name">Span name.</param>
         /// <param name="kind">Kind.</param>
-        /// <param name="attributes">Initial attributes for the span.</param>
+        /// <param name="activityTagsCollection">Initial attributes for the span.</param>
```

Comment:
we should call the parameter `initialAttributes`

Reply:
done